client/packages/plugins/backendTypes/generated/AverageMonthlyConsumptionItem.ts

```diff
@@ -0,0 +1,3 @@
+// This file was generated by [ts-rs](https://github.com/Aleph-Alpha/ts-rs). Do not edit this file manually.
+
+export type AverageMonthlyConsumptionItem = { average_monthly_consumption: number | null, };
```

Comment:
these are auto generated, i haven't written up any docs about back end plugins, this basically comes from, [derive(TS](https://github.com/msupply-foundation/open-msupply/blob/73289fc25807543f164900020d284e9f6b2a6697/server/service/src/backend_plugin/types/amc.rs#L7) and then: https://github.com/msupply-foundation/open-msupply/blob/73289fc25807543f164900020d284e9f6b2a6697/server/service/src/backend_plugin/types/mod.rs#L3-L26

Reply:
> i haven't written up any docs about back end plugins

are they coming? 🥺